stl/inc/vector

```diff
@@ -1043,6 +1043,12 @@ public:
         pointer& _Myfirst = _My_data._Myfirst;
         pointer& _Mylast  = _My_data._Mylast;
 
+#if _ITERATOR_DEBUG_LEVEL == 2
+        auto _Valptr = _Unfancy(_STD addressof(_Val));

```

Comment:
`addressof` always returns a raw pointer (as it's just `&_Val` that's immune to `operator&` overloads), so `_Unfancy` is unnecessary. In contrast, `_Myfirst` and `_Mylast` are potentially-fancy pointers obtained from the container's allocator.

Also, this pointer can be top-level `const`.

```suggestion
        const auto _Valptr = _STD addressof(_Val);
```

Reply:
@StephanTLavavej 
Sorry, I didn't commit your suggestion in Github just to prevent unnecessary CI build and test runs. However, I apply this suggestion as part of my commit. Thank you.